examples/ocm-partners/users-ailleron.json

```diff
@@ -13,7 +13,7 @@
 	{
 		"id": {
 			"opaque_id": "mateusz5678",
-			"idp": "https://ailleron.com"
+			"idp": "ailleron.com"
```

Comment:
Hi @ishank011 thanks for updating the configs! There is just one small issue, Ailleron uses `softwaremind.com` domain in GOCDB now.

Reply:
@mirekys Will fix it. Thanks!